### ISSUE TITLE
June 2023 Endgame

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#186203]
- June 26 Code freeze for the endgame
- June 30 [Endgame done](https://github.com/Microsoft/vscode/wiki/Development-Process#inside-an-iteration)
- July 6 Expected release date (this may change)

##### Monday
- [x] Check that all queries in this issue use the current milestone @chrmarti 
- [x] Run [OSS tool](https://github.com/microsoft/vscode-build-tools/tree/main/distro-tools) @chrmarti 
- [x] Update links in the [Endgame notebooks](https://github.com/microsoft/vscode/blob/main/.vscode/notebooks/endgame.github-issues) to point to new milestone @chrmarti 
- [x] Code freeze at 5pm PT
- [x] Ensure we have a green build on all platforms at 5pm PT
- [x] [🔖All closed feature-requests](https://github.com/issues?q=-label%3Aon-testplan%20repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-copilot%20repo%3Amicrosoft%2Fvscode-copilot-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels%20milestone%3A%22June%202023%22%20is%3Aissue%20is%3Aclosed%20reason%3Acompleted%20label%3Averification-needed%20-label%3Averified) either have a `verification-needed` or `on-testplan` label
- [x] Create test plan items following the template [here](https://github.com/microsoft/vscode/wiki/Writing-Test-Plan-Items) by 6pm PT
- [x] Update your availability for testing here - https://tools.code.visualstudio.com/team-manifest **team**
  - [x] Update availability of testers in vacation (Check OOF section in the Internal Backlog Plan). Double check N/A testers. @chrmarti
  - [x] Remind team that in the event of sickness or overload with TPIs, to inform the endgame champ ASAP so items can be reassigned

##### Tuesday
- [x] Test plan items assigned (using https://tools.code.visualstudio.com/test-plan-items)
  - Run the tool multiple times to balance load if test items come in later and assignments are already made
  - [Assigned to you](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+assignee%3A%40me++label%3Atestplan-item+)
- [x] Test build starts at 7am CET
- [x] Test plan ready by 8am CET
- [x] Remind the team about the priorities
  - Tuesday should be dedicated exclusively to testing activities. Our goal is to ensure the completion of all Test Plan Items (TPIs) and subsequently proceed with the verification phase. Fixes or commits should be refrained from, unless there are exceptional circumstances such as blocked TPIs or build-related issues
  - Redmond members commence testing the TPIs authored by Zurich members, enabling them to address any feedback on the following day.
- [x] [🔖Testing](https://github.com/issues?q=repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-copilot%20repo:microsoft/vscode-copilot-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-python-debugger%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels%20is:issue%20is:open%20label:testplan-item)
- [x] [🔖Verification needed](https://github.com/issues?q=-label%3Aon-testplan%20repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-copilot%20repo%3Amicrosoft%2Fvscode-copilot-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels%20milestone%3A%22June%202023%22%20is%3Aissue%20is%3Aclosed%20reason%3Acompleted%20label%3Averification-needed%20-label%3Averified)

##### Wednesday
- [x] [🔖Testing](https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3Atestplan-item+repo%3Amicrosoft%2Fvscode+repo%3Amicrosoft%2Fvscode-internalbacklog+repo%3Amicrosoft%2Fvscode-dev+repo%3Amicrosoft%2Fvscode-remote-repositories-github+repo%3Amicrosoft%2Fvscode-remote-release+repo%3Amicrosoft%2Fvscode-js-debug+repo%3Amicrosoft%2Fvscode-pull-request-github+repo%3Amicrosoft%2Fvscode-livepreview+repo%3Amicrosoft%2Fvscode-python+repo%3Amicrosoft%2Fvscode-python-debugger+repo%3Amicrosoft%2Fvscode-jupyter+repo%3Amicrosoft%2Fvscode-jupyter-internal+repo%3Adevcontainers%2Fcli+repo%3Adevcontainers%2Fci+repo%3Amicrosoft%2Fvscode-hexeditor)
  - These remaining TPIs should be the ones that are currently blocked. Discuss during the standup and redistrubute assignments based on the TPI owner and the test coverage. For instance, if a TPI is owned by a member from Zurich and has not undergone sufficient testing, it will be reassigned to one of the Zurich team members.
- [x] Remind team members to triage issues found in testing and assign major issues that they intend to fix to the current milestone
- [x] [🔖Verification needed](https://github.com/issues?q=-label%3Aon-testplan%20repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-copilot%20repo%3Amicrosoft%2Fvscode-copilot-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels%20milestone%3A%22June%202023%22%20is%3Aissue%20is%3Aclosed%20reason%3Acompleted%20label%3Averification-needed%20-label%3Averified)

##### Thursday
- [x] Make sure to have the `insider` build @alexr00
- [x] Pause scheduled `insider` builds (includes vscode.dev publishing) @alexr00
- [x] Fixing (self-assigned, milestone assigned)
  - Increased scrutiny sets in due to testing being completed. Fixes pose a much higher risk
  - Move issues to the next month that can be deferred
- [x] [🔖Verification needed](https://github.com/issues?q=is%3Aissue%20milestone%3A%22June%202023%22%20repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels%20-label%3Aiteration-plan%20-label%3Aendgame-plan%20-label%3Atestplan-item%20label%3Averification-needed%20-label%3Averified%20is%3Aclosed%20-label%3Aon-testplan)
- [x] [🔖Verification](https://github.com/issues?q=repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-copilot%20repo:microsoft/vscode-copilot-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-python-debugger%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels%20milestone:%22June%202023%22%20is:issue%20is:closed%20reason:completed%20%20label:bug%20-label:verified%20-label:on-testplan%20-label:*duplicate%20-label:duplicate%20-label:invalid%20-label:*as-designed%20-label:error-telemetry%20-label:verification-steps-needed%20-label:z-author-verified%20-label:unreleased%20sort:updated-asc)

##### Friday
- [x] Branch & Release by latest 9AM CET @alexr00
  - [x] Branch following repositories to `release/<x.y>`
    - [x] vscode
    - [x] vscode-distro
    - [x] vscode-dev
  - [x] Announce `main` is open for business
  - [x] Build and manually release Insider from release/<x.y> branch for Code
  - [x] Build and manually release Insider from release/<x.y> branch for vscode.dev (if required)
- [x] Build a `stable` build from release/<x.y> branch to ensure stable build is green @alexr00
- [x] Bump up the version in `package.json` on `main` - @alexr00
- [x] Localization: Run [Update VS Code Branch](https://monacotools.visualstudio.com/Monaco/_build?definitionId=369) build with `release/*` as the VS Code Branch parameter (it's the default so you shouldn't have to change anything) @alexr00
- [x] [Create next milestone](https://github.com/microsoft/vscode-internalbacklog/milestones/new) and ensure that it has a due date. The created milestone and its due date will be automatically synced across our repos @alexr00
- Satellite modules/npm packages ready, version updated, smoke tested
  - [x] yo generator **@aeschli**
  - [x] js-debug **@connor4312**
  - [x] Debug Adapter Protocol **@connor4312**
  - [x] vscode-debugadapter-node **@connor4312**
  - [x] Mock Debug **@roblourens**
  - [x] vscode-test-web **@aeschli**
  - [x] vsce **@joaomoreno**
- [x] Only candidate issues are open and assigned to [🔖milestone](https://github.com/issues?q=is%3Aopen%20is%3Aissue%20milestone%3A%22June%202023%22%20repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels)
- [x] All issues [🔖verified](https://github.com/issues?q%253Dis%253Aissue%2520%2520repo%253Amicrosoft%252Fvscode%2520repo%253Amicrosoft%252Fvscode-remote-release%2520repo%253Amicrosoft%252Fvscode-js-debug%2520repo%253Amicrosoft%252Fvscode-pull-request-github%2520repo%253Amicrosoft%252Fvscode-github-issue-notebooks%2520repo%253Amicrosoft%252Fvscode-internalbacklog%2520repo%253Amicrosoft%252Fvscode-dev%2520repo%253Amicrosoft%252Fvscode-unpkg%2520repo%253Amicrosoft%252Fvscode-references-view%2520repo%253Amicrosoft%252Fvscode-anycode%2520repo%253Amicrosoft%252Fvscode-hexeditor%2520repo%253Amicrosoft%252Fvscode-extension-telemetry%2520repo%253Amicrosoft%252Fvscode-livepreview%2520repo%253Amicrosoft%252Fvscode-remotehub%2520repo%253Amicrosoft%252Fvscode-settings-sync-server%2520repo%253Amicrosoft%252Fvscode-remote-repositories-github%2520repo%253Amicrosoft%252Fmonaco-editor%2520repo%253Amicrosoft%252Fvscode-vsce%2520repo%253Amicrosoft%252Fvscode-dev-chrome-launcher%2520repo%253Amicrosoft%252Fvscode-emmet-helper%2520repo%253Amicrosoft%252Fvscode-livepreview%2520repo%253Amicrosoft%252Fvscode-livepreview%2520repo%253Amicrosoft%252Fvscode-python%2520repo%253Amicrosoft%252Fvscode-python-debugger%2520repo%253Amicrosoft%252Fvscode-jupyter%2520repo%253Amicrosoft%252Fvscode-jupyter-internal%2520repo%253Amicrosoft%252Fvscode-github-issue-notebooks%2520repo%253Amicrosoft%252Fvscode-l10n%2520repo%253Amicrosoft%252Fvscode-remote-tunnels%2520-label%253Aiteration-plan%2520-label%253Aendgame-plan%2520milestone%253A%2522June%25202023%2522%2520-label%253Atestplan-item%2520label%253Abug%2520-label%253Aduplicate%2520-label%253A%252Aduplicate%2520-label%253Ainvalid%2520-label%253Aas-designed%2520-label%253Aerror-telemetry%2520is%253Aclosed%2520-label%253Averified%2520-label%253Aon-testplan)
- [x] All open PRs on the milestone [🔖merged or deferred](https://github.com/issues?q%253Drepo%253Amicrosoft%252Fvscode%2520repo%253Amicrosoft%252Fvscode-remote-release%2520repo%253Amicrosoft%252Fvscode-js-debug%2520repo%253Amicrosoft%252Fvscode-pull-request-github%2520repo%253Amicrosoft%252Fvscode-github-issue-notebooks%2520repo%253Amicrosoft%252Fvscode-internalbacklog%2520repo%253Amicrosoft%252Fvscode-dev%2520repo%253Amicrosoft%252Fvscode-unpkg%2520repo%253Amicrosoft%252Fvscode-references-view%2520repo%253Amicrosoft%252Fvscode-anycode%2520repo%253Amicrosoft%252Fvscode-hexeditor%2520repo%253Amicrosoft%252Fvscode-extension-telemetry%2520repo%253Amicrosoft%252Fvscode-livepreview%2520repo%253Amicrosoft%252Fvscode-remotehub%2520repo%253Amicrosoft%252Fvscode-settings-sync-server%2520repo%253Amicrosoft%252Fvscode-remote-repositories-github%2520repo%253Amicrosoft%252Fmonaco-editor%2520repo%253Amicrosoft%252Fvscode-vsce%2520repo%253Amicrosoft%252Fvscode-dev-chrome-launcher%2520repo%253Amicrosoft%252Fvscode-emmet-helper%2520repo%253Amicrosoft%252Fvscode-livepreview%2520repo%253Amicrosoft%252Fvscode-livepreview%2520repo%253Amicrosoft%252Fvscode-python%2520repo%253Amicrosoft%252Fvscode-python-debugger%2520repo%253Amicrosoft%252Fvscode-jupyter%2520repo%253Amicrosoft%252Fvscode-jupyter-internal%2520repo%253Amicrosoft%252Fvscode-github-issue-notebooks%2520repo%253Amicrosoft%252Fvscode-l10n%2520repo%253Amicrosoft%252Fvscode-remote-tunnels%2520-label%253Aiteration-plan%2520-label%253Aendgame-plan%2520milestone%253A%2522June%25202023%2522%2520%2520is%253Aopen%2520is%253Apr%2520)
- [x] Fixing (PR + review required - major bugs only - to be discussed in stand-up meeting, labeled as `candidate`)
- [x] All release notes updated
  - release notes are collected in a file named *`v<Major>_<Minor>.md`* in this [repo directory](https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/)
- [x] Acknowledge pull requests in release notes. We acknowledge PRs from outside the team. We have improved [the tooling](https://tools.code.visualstudio.com/acknowledgement) so that the endgame champion can generate the pull request acknowledgment for all repositories at once. @chrmarti
  - [x] `debug-adapter-protocol`, `inno-updater`, `jsonc-parser`, `language-server-protocol`, `lsif-node`, `vscode`, `vscode-codicons`, `vscode-css-languageservice`, `vscode-debugadapter-node`, `vscode-dev-containers`, `vscode-docs`, `vscode-emmet-helper`, `vscode-eslint`, `vscode-extension-samples`, `vscode-generator-code`, `vscode-hexeditor`, `vscode-html-languageservice`, `vscode-js-debug`, `vscode-js-debug-companion`, `vscode-js-profile-visualizer`, `vscode-jshint`, `vscode-json-languageservice`, `vscode-languageserver-node`, `vscode-livepreview`, `vscode-loader`, `vscode-lsif-extension`, `vscode-node-debug`, `vscode-node-debug2`, `vscode-pull-request-github`, `vscode-recipes`, `vscode-references-view`, `vscode-textmate`, `vscode-vsce`
- [x] Acknowledge [issue trackers](https://github.com/microsoft/vscode-internalbacklog/wiki/Community-Triage---Credits) from the community **@alexr00**
- When done fixing/verifying and there are changes since last build at the end of day PT
  - [x] Build and manually release Insider from release/<x.y> for Code @alexr00
  - [x] Build and manually release Insider from release/<x.y> for vscode.dev (if required) @alexr00

##### Friday/Monday
- [x] Polish release notes **redmond**
- [x] Fixing (only critical bugs - no string changes)

##### Monday - Wednesday
- [x] [🔖milestone issues](https://github.com/issues?q=repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels%20-label%3Aiteration-plan%20-label%3Aendgame-plan%20milestone%3A%22June%202023%22%20%20is%3Aopen%20is%3Aissue)
- [x] [🔖candidate issues](https://github.com/issues?q=repo%3Amicrosoft%2Fvscode%20repo%3Amicrosoft%2Fvscode-remote-release%20repo%3Amicrosoft%2Fvscode-js-debug%20repo%3Amicrosoft%2Fvscode-pull-request-github%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20repo%3Amicrosoft%2Fvscode-internalbacklog%20repo%3Amicrosoft%2Fvscode-dev%20repo%3Amicrosoft%2Fvscode-unpkg%20repo%3Amicrosoft%2Fvscode-references-view%20repo%3Amicrosoft%2Fvscode-anycode%20repo%3Amicrosoft%2Fvscode-hexeditor%20repo%3Amicrosoft%2Fvscode-extension-telemetry%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-remotehub%20repo%3Amicrosoft%2Fvscode-settings-sync-server%20repo%3Amicrosoft%2Fvscode-remote-repositories-github%20repo%3Amicrosoft%2Fmonaco-editor%20repo%3Amicrosoft%2Fvscode-vsce%20repo%3Amicrosoft%2Fvscode-dev-chrome-launcher%20repo%3Amicrosoft%2Fvscode-emmet-helper%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-livepreview%20repo%3Amicrosoft%2Fvscode-python%20repo%3Amicrosoft%2Fvscode-python-debugger%20repo%3Amicrosoft%2Fvscode-jupyter%20repo%3Amicrosoft%2Fvscode-jupyter-internal%20repo%3Amicrosoft%2Fvscode-github-issue-notebooks%20label%3Acandidate%20repo%3Amicrosoft%2Fvscode-l10n%20repo%3Amicrosoft%2Fvscode-remote-tunnels%20-label%3Aiteration-plan%20-label%3Aendgame-plan%20milestone%3A%22June%202023%22%20%20is%3Aopen%20is%3Aissue)
- [x] Polish release notes **redmond**
- [x] Cherry-pick hand-picked and reviewed changes to `release/<x.y>` @alexr00
- [x] Build `Insider` from `release/<x.y>` @alexr00
- [x] Manually release `Insider` @alexr00
- [x] Build stable for all platforms as new candidate issues come in @alexr00
- [x] Documentation updated
- [x] Run `scripts/test-documentation.sh|bat` and add file or fix issues if there are new colors that are not documented. @alexr00

> **Note:** The `Insiders` build needs to be in the wild for 24 hours before we can enter the last phase of the endgame. @alexr00

##### Wednesday/Thursday - Expected release day (this may change)
- [x] Build stable for all platforms @alexr00
- [x] [Sanity Testing](https://github.com/microsoft/vscode/wiki/Sanity-Check)
  - [x] Windows 32 bit @hediet
    - [x] System Installer
    - [x] User Installer
    - [x] Archive
    - [x] Server
    - [x] CLI
  - [x] Windows 64 bit @hediet
    - [x] System Installer
    - [x] User Installer
    - [x] Archive
    - [x] Server
    - [x] CLI
  - [x] Windows ARM64 @joaomoreno
    - [x] System Installer
    - [x] User Installer
    - [x] Archive
    - [x] CLI
  - [x] macOS Intel @isidorn
    - [x] Archive
    - [x] Universal Archive
    - [x] Server
    - [x] CLI
  - [x] macOS ARM64 @jrieken
    - [x] Archive
    - [x] Universal Archive
    - [x] Server
    - [x] CLI
  - [x] Linux x64 @chrmarti
    - [x] Archive
    - [x] DEB
    - [x] RPM
    - [x] Snap
  - [x] Linux Server @sandy081 _(can be tested from any OS with Docker)_
    - [x] x64
    - [x] ARM32
    - [x] ARM64
    - [x] x64 Alpine
    - [x] ARM64 Alpine
  - [x] Linux CLI @ulugbekna _(can be tested from any OS with Docker)_
    - [x] x64
    - [x] ARM32
    - [x] ARM64
    - [x] x64 Alpine
    - [x] ARM64 Alpine
- [x] Publish website **@gregvanl**
- [x] Publish to stable @meganrogge 
- [x] Trigger the [vscode.dev Deploy pipeline](https://dev.azure.com/monacotools/Monaco/_build?definitionId=276) for the `prod` deployment target. @meganrogge 
  - [x] Request approval from another team member at the necessary step to deploy the vscode.dev build. @meganrogge 
- [x] Create an official release @meganrogge 
  - [x] Create a tag (make sure you pull the release branch first): `git tag <x.y.z>`
  - [x] Push the tag: `git push origin <x.y.z>`
  - [x] Create a GitHub release: [Open the GitHub tags](https://github.com/microsoft/vscode/tags), and click far right `... > Create Release`. Use the correct title and description from our release notes. Also change the relative links for the key highlight list items to absolute links [Example](https://github.com/microsoft/vscode/releases/tag/1.39.0)
- [x] Twitter announcement **@chrisdias**
- [x] Enable scheduled `insider` builds @meganrogge 
- [x] [Publish @types/vscode](https://github.com/microsoft/vscode/wiki/Publish-vscode-types) @meganrogge 
- [x] Close the milestone on [GitHub](https://github.com/microsoft/vscode/milestones) @meganrogge 
- [x] If no Patch Tuesday release is planned, [let the Updates team know of that fact](https://github.com/microsoft/vscode-internalbacklog/wiki/Patch-Tuesday-E-Mail-Template) ([CSEClientPatching@microsoft.com](mailto:CSEClientPatching@microsoft.com)) @alexr00
- [x] If a Recovery or Patch Tuesday release is needed proceed to running [Recovery Release](https://github.com/microsoft/vscode/wiki/Running-the-Endgame#recovery-release) or [Patch Tuesday Release](https://github.com/microsoft/vscode-internalbacklog/wiki/Handling-MSRC-Cases) @alexr00
